src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,338 @@
+        ),
+        max_length=1000,
+        blank=True,
+    )
+    soort_partij = models.CharField(
+        _("Soort partij"),
+        help_text=_("Geeft aan van welke specifieke soort partij sprake is."),
+        max_length=14,
+        choices=SoortPartij.choices,
+        blank=False,
+    )
+    indicatie_geheimhouding = models.BooleanField(
+        _("Indicatie geheimhouding"),
+        help_text=_(
+            "Geeft aan of de verstrekker van partijgegevens heeft aangegeven dat "
+            "deze gegevens als geheim beschouwd moeten worden."
+        ),
+        blank=False,
+    )
+    voorkeurskanaal = models.CharField(
+        _("Voorkeurskanaal"),
+        help_text=_(
+            "Kanaal dat de partij bij voorkeur gebruikt voor contact met de gemeente."
+        ),
+        max_length=50,
+        blank=True,
+    )
+    # TODO: change max_length to more realistic number
+    voorkeurstaal = models.CharField(
+        _("Voorkeurstaal"),
+        help_text=_(
+            "Taal waarin de partij bij voorkeur contact heeft met de gemeente."
+        ),
+        max_length=255,
+        blank=True,
+    )
+    indicatie_actief = models.BooleanField(
+        _("Indicatie actief"),
+        help_text=_(
+            "Geeft aan of de contactgegevens van de partij nog gebruikt morgen worden om contact op te nemen. "
+            "Gegevens van niet-actieve partijen mogen hiervoor niet worden gebruikt."
+        ),
+        blank=False,
+    )
+
+    # Correspondentieadres model fields:
+    correspondentieadres_nummeraanduiding_id = models.UUIDField(
+        default=uuid.uuid4,
+        unique=True,
+        max_length=40,
+        help_text=_(
+            "Identificatie van het adres bij de Basisregistratie Adressen en Gebouwen."
+        ),
+    )
+    correspondentieadres_adresregel1 = models.CharField(
+        _("Adresregel 1"),
+        help_text=_(
+            "Eerste deel van het adres dat niet voorkomt in de Basisregistratie Adressen en Gebouwen."
+        ),
+        max_length=80,
+        blank=True,
+    )
+    correspondentieadres_adresregel2 = models.CharField(
+        _("Adresregel 2"),
+        help_text=_(
+            "Tweede deel van het adres dat niet voorkomt in de Basisregistratie Adressen en Gebouwen."
+        ),
+        max_length=80,
+        blank=True,
+    )
+    correspondentieadres_adresregel3 = models.CharField(
+        _("Adresregel 3"),
+        help_text=_(
+            "Derde van het adres dat niet voorkomt in de Basisregistratie Adressen en Gebouwen."
+        ),
+        max_length=80,
+        blank=True,
+    )
+    correspondentieadres_land = models.IntegerField(
+        _("Land"),
+        help_text=_(
+            "Een code, opgenomen in Tabel 34, Landentabel, die het land (buiten Nederland) "
+            "aangeeft alwaar de ingeschrevene verblijft."
+        ),
+        validators=[MaxValueValidator(9999)],
+        blank=True,
+    )
```

Comment:
I dont really know if in the specs the `Referentielijst Land` corresponds to this field, but there it says that the landcode should be a string of 4 digits (so not an int).

But if it needs to remain an int, you also need to add `null=True`.

Reply:
I don't know forsure if `referentielijst land` corresponds either, but I did check the Tabel 34 Landentable, and evey code is made out off 4 digits
![Screenshot from 2023-10-12 18-01-36](https://github.com/maykinmedia/open-klant/assets/101265650/80da4840-8b61-434c-8c43-318d3f33444c)